idl/raft_storage.idl.hh

```diff
@@ -20,6 +20,11 @@ struct tagged_id {
     utils::UUID id;
 };
 
+template<typename Tag>
+struct tagged_uint64 {
+    uint64_t value();
+};
+
```

Comment:
How did we change something in idl/ and not think about compatibility?

Reply:
I apologize, my bad.
I wonder if we can add a simple uograde test to CI to prevent similar regressions.

Reply:
It's hard because CI only sees the current state.

We need to generate some kind of signature and commit it, and compare the existing signature to all past signatures for compatibility.

Reply:
sounds feasible, like a schema version for idl